core/store/src/lib.rs

```diff
@@ -270,6 +273,43 @@ pub fn create_store(path: &str) -> Arc<Store> {
     Arc::new(Store::new(db))
 }
 
+fn get_all_block_hash_to_outcomes(store: &Store, block_hash: &CryptoHash) -> HashSet<CryptoHash> {
+    match store.get_ser(ColOutcomesByBlockHash, block_hash.as_ref()) {
+        Ok(Some(hash_set)) => hash_set,
+        Ok(None) => HashSet::new(),
+        Err(e) => panic!("Can't read DB, {:?}", e),
+    }
+}
+
+pub fn fill_col_outcomes_by_hash(store: &Store) {
```

Comment:
Should we create a separate file for the migration utils? It seems that this method is only useful once, when we want to migrate the DB.

Reply:
Makes sense.

Reply:
++